modules/core/client/app/init.js

```diff
@@ -30,7 +30,9 @@ angular.module(ApplicationConfiguration.applicationModuleName).run(function ($ro
         if (Authentication.user !== undefined && typeof Authentication.user === 'object') {
           $state.go('forbidden');
         } else {
-          $state.go('authentication.signin');
+          $state.go('authentication.signin').then(function () {
```

Comment:
Yes @ilanbiala Ok, in fact that depends on the audience and the application, so we shouldn't worry about it here.